packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/CodeMirror.tsx

```diff
@@ -739,8 +739,7 @@ function CodeMirror(props: NoteBodyEditorProps, ref: any) {
 		}
 
 		async function onContextMenu(_event: any, params: any) {
-			if (!pointerInsideEditor(params.x, params.y)) return;
-
+			if (!pointerInsideEditor(params.x, params.y) || !params.isEditable || params.inputFieldType !== 'none') return;
```

Comment:
Just move all that logic inside `pointerInsideEditor` and pass the `params` object to it. Add a comment that explains why you are checking isEditable and inputFieldType.

Don't mark my comment as "resolved".

Reply:
Done :)